packages/editor/src/App.css

```diff
@@ -28,6 +29,9 @@
 
   --selected-text: #ffffff;
   --background-radio-checkbox: var(--A50);
+  --icon-fill-radio-checkbox: var(--A300);
+
+  --selectedRowFont: var(--background);
```

Comment:
please use constistent names -> --selected-row-font

And please name the variables like the content what they include, font is not the color for me, it is the font.
So this should be something like --selected-row-color (because color is the color of the font)